src/components/styles/Dishes.css

```diff
@@ -0,0 +1,84 @@
+@import url(https://fonts.googleapis.com/css?family=Open+Sans);
+@import url('https://fonts.cdnfonts.com/css/lato');
+
+@media only screen and (max-width: 768px) {
+  .wrapper {
+    background-image: url('/src/assets/bg4.jpg');
+    border-bottom: 6px solid rgb(80, 80, 78);
+  }
+
+  .dish-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-gap: 10px;
+    margin: 2rem 1rem 1rem;
+  }
```

Comment:
- [ ] [REQUIRED] Kindly note that the design template provided does not have a border around the images on the landing/home page. Please remove all the white borders on your design and ensure that each image has a color of a similar background as it is in the design template, by doing this you will be sticking to the design guidelines.
- To archive this, you may want to remove the background image and replace it with a background color. let's do the same for the details page as well.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="300" alt="Screenshot 2023-04-27 at 18 42 15" src="https://user-images.githubusercontent.com/58692590/234946926-a64faad2-fb84-4a1a-b940-c5e419302d97.png"> | <img width="245" alt="Screenshot 2023-04-27 at 18 25 17" src="https://user-images.githubusercontent.com/58692590/234947042-5cf3593b-f6e8-4d64-914f-69359a2e1253.png">